### PR TITLE
Added improvment to the cursor

### LINKED ISSUE #60: Change mouse cursor to a pointer when hovering over "show forks"
## Aceptance criteria
- [ ] Cursor is now a pointer or clicker when hovering over "show forks"

### PR DESCRIPTION
**Pull request by te4-andre-skvarc**
## What issue did you implement or fix?
Fixed #60 

## Summary
Added hover function to the cursor

### CODE REVIEW

**Review by te4-tintin-wihlborg — APPROVED** [review]
Looks good

File: public/css/main.css [hunk 1]
```diff
@@ -14,3 +14,7 @@ h3 {
 .row .col.s4 {
   height: 17em !important;
 }
+
+.repoForks{
```
Comment (te4-adrian-almetun-smeds) [inline public/css/main.css:18]: Due to linting errors.
```suggestion
.repoForks {
```

File: public/css/main.css [hunk 2]
```diff
@@ -14,3 +14,7 @@ h3 {
...
+    cursor:pointer;
```
Comment (te4-adrian-almetun-smeds) [inline public/css/main.css:19]: Linter here as well.
```suggestion
    cursor: pointer;
```

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Nice feature, just do some linting.

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
Nice 👍

**Review by te4-david-jensen — APPROVED** [review]
Looks good! 👍 :shipit:

**Pull request merged**